folium/features.py

```diff
@@ -184,22 +184,56 @@ def __init__(self, data):
 
                     # providing string
                     GeoJson(open('foo.json').read())
+            style_function: function, default None
```

Comment:
I think we should have some defaults and [simplestyle-spec](https://github.com/mapbox/simplestyle-spec/tree/master/1.1.0) is a good candidate.